kolena/workflow/metrics/_plots.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2021-2023 Kolena Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from collections import defaultdict
+from typing import Dict
+from typing import List
+from typing import Optional
+from typing import Set
+
+from kolena._utils import log
+from kolena.workflow.evaluator import ConfusionMatrix
+from kolena.workflow.metrics._geometry import MulticlassInferenceMatches
```

Comment:
Should `MulticlassInferenceMatches` be available under `kolena.workflow.metrics`?  It's currently missing from the API doc.

Reply:
nice catch